src/Antd_DatePicker.re

```diff
@@ -152,7 +157,7 @@ module RangePicker = {
       ~disabled: Js.boolean=?,
       ~allowClear: Js.boolean=?,
       ~popupStyle: ReactDOMRe.Style.t=?,
-      ~locale: 'c=?,
+      ~locale: 'j=?,
```

Comment:
It's really hard to use alphabet and keeping track of all usage and make sure they are unique like this. You can use a longer type parameter like `'locale`